.github/workflows/algolia-scraper.yml

```diff
@@ -0,0 +1,25 @@
+name: algolia-scraper
+on:
+  push: # only run on push events
+    branches:
+      - main
+  check_suite: # only run check suite events
+    types: [completed]
+  workflow_dispatch: # only run on workflow dispatch events
```

Comment:
What is this for?  I've never used it.  Why do we need it?

Reply:
CI to run the index generation?